packages/checkout/widgets-lib/src/views/top-up/TopUpView.tsx

```diff
@@ -27,14 +29,15 @@ import {
   getOnRampFeeEstimation,
 } from '../../lib/feeEstimation';
 import { CryptoFiatActions, CryptoFiatContext } from '../../context/crypto-fiat-context/CryptoFiatContext';
-import { ConnectLoaderContext } from '../../context/connect-loader-context/ConnectLoaderContext';
 import { isPassportProvider } from '../../lib/providerUtils';
 import { OnRampWidgetViews } from '../../context/view-context/OnRampViewContextTypes';
 import { EventTargetContext } from '../../context/event-target-context/EventTargetContext';
 import { TopUpMenuItem } from './TopUpMenuItem';
 
 interface TopUpViewProps {
   widgetEvent: IMTBLWidgetEvents,
+  checkout?: Checkout,
+  provider?: Web3Provider,
```

Comment:
Passing through via props now instead as the TopUpView was using the values from connect loader state, which the move widget does not have